server/data/scanner.js

```diff
@@ -112,18 +114,27 @@ scanner = () => {
 
           // If the 2 values aren't equal then the lines crossed, If they are both true or both false the lines did not cross
           if (!x === y) {
+            // this is the only way I could get the timing to work
+            // refactor this at some point to increase reliability
```

Comment:
Yea this is a bit hard to understand why not have it in one Axios call and do then()... Because you only getting just the uid and phone number in the previous call you could add this axios call in then() statement and get the alpha vantage. 
But overall I believe I understand what it trying to do because of your comments.

Reply:
i'm not sure i follow. Do you mean: when i get the favorite i should also get the alpha vantage info?
That should cut down and simplify the code i guess. I will try it. Seems like it would keep all the data more together that way.  thanks,